KWZP2019/KWZP2019/Finances/ProfitsAndFinancialBalance.cs

```diff
@@ -19,86 +19,45 @@ public partial class Profits : Form
-            dgvProfits.Columns[1].HeaderText = "Nazwa klienta";
-            dgvProfits.Columns[2].HeaderText = "Data zamówienia";
-            dgvProfits.Columns[3].HeaderText = "Wartość zamówienia";
-            dgvProfits.AutoResizeColumns(DataGridViewAutoSizeColumnsMode.AllCellsExceptHeader);
             dgvInvoices.DataSource = db.vExpencesInvoices.ToList();
-            this.dgvInvoices.Columns["IdInvoice"].Visible = false;
-            dgvInvoices.Columns[1].HeaderText = "Data wystawienia";
-            dgvInvoices.Columns[2].HeaderText = "Wartość faktury";
-            dgvInvoices.Columns[3].HeaderText = "Nazwa podmiotu";
-            dgvInvoices.Columns[4].HeaderText = "Typ faktury";
-            dgvInvoices.AutoResizeColumns(DataGridViewAutoSizeColumnsMode.AllCellsExceptHeader);
             dgvOrders.DataSource = db.vExpencesOrders.ToList();
-            this.dgvOrders.Columns["IdSfOrder"].Visible = false;
-            dgvOrders.Columns[1].HeaderText = "Wartość faktury";
-            dgvOrders.Columns[2].HeaderText = "Data wystawienia";
-            dgvOrders.Columns[3].HeaderText = "Nazwa podmiotu";
-            dgvOrders.AutoResizeColumns(DataGridViewAutoSizeColumnsMode.AllCellsExceptHeader);
             dgvOutsourcing.DataSource = db.vExpencesOutsourcings.ToList();
-            this.dgvOutsourcing.Columns["IdOutsourcing"].Visible = false;
-            dgvOutsourcing.Columns[1].HeaderText = "Wartość";
-            dgvOutsourcing.Columns[2].HeaderText = "Data";
-            dgvOutsourcing.Columns[3].HeaderText = "Nazwa podmiotu";
-            dgvOutsourcing.AutoResizeColumns(DataGridViewAutoSizeColumnsMode.AllCellsExceptHeader);
             dgvSalaries.DataSource = db.vExpencesPayments.ToList();
-            this.dgvSalaries.Columns["IdPayment"].Visible = false;
-            dgvSalaries.Columns[1].HeaderText = "Podstawa wynagrodzenia";
-            dgvSalaries.Columns[2].HeaderText = "Data wypłaty";
-            dgvSalaries.Columns[3].HeaderText = "Premia";
-            dgvSalaries.Columns[4].HeaderText = "Imię";
-            dgvSalaries.Columns[5].HeaderText = "Nazwisko";
-            dgvSalaries.AutoResizeColumns(DataGridViewAutoSizeColumnsMode.AllCellsExceptHeader);
+            dgvPartOrder.DataSource = db.vExpencePartOrders.ToList();
         }
         private void btSum_Click(object sender, EventArgs e)
         {
             dgvProfits.DataSource = db.vIncomesProfits.ToList().Where(vIncomesProfits =>
-            vIncomesProfits.OrderDate > dtpStartDate.Value && vIncomesProfits.OrderDate < dtpEndDate.Value)
-            .Select(vIncomesProfits => new { vIncomesProfits.IdCustomer, vIncomesProfits.CustomerName, vIncomesProfits.OrderDate, vIncomesProfits.Cost })
-            .ToList();
-            dgvProfits.AutoResizeColumns(DataGridViewAutoSizeColumnsMode.AllCellsExceptHeader);
+            vIncomesProfits.OrderDate > dtpStartDate.Value && vIncomesProfits.OrderDate < dtpEndDate.Value).ToList();
```

Comment:
czy tylko dla mnie kod teraz jest duuużo czytelniejszy?

Reply:
Faktycznie, ale kiedy kod jest czytelny dla każdego, to już nie jesteśmy niezastąpieni w firmie :)